bin/bst-intend.ts

```diff
@@ -77,6 +77,12 @@ program
 
             try {
                 speaker.intended(intentName, slots, function(error: any, response: any, request: any) {
+                    if (error !== undefined && error !== null) {
```

Comment:
@jperata, I think it would be cleaner to have:

```
if (error) {
console.log("Intended: " + intentName);
console.log("");
console.log("Error: " + error.message);
return;
}
```